src/router/index.js

```diff
@@ -1,15 +1,18 @@
 import Vue from 'vue';
 import VueRouter from 'vue-router';
 import AdminView from '../views/AdminView.vue';
-import UserView from '../views/UserView.vue';
+import UserAvailableResView from '../views/UserAvailableResView.vue';
+import UserConfirmResView from '../views/UserConfirmResView.vue';
+import UserResView from '../views/UserResView.vue';
+import UserReviewResView from '../views/UserReviewResView.vue';
```

Comment:
Adding a view for every component is redundant. Only two views should exist and redirects to their components will exist in them.